.github/workflows/upstream-check.yml

```diff
@@ -0,0 +1,56 @@
+name: Upstream
+
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened, ready_for_review ]
+
+jobs:
+  upstream_run:
+    name: ${{ matrix.repo }} (dry-run)
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    concurrency:
+      group: upstream-check-${{github.event_name}}-${{ matrix.repo }}-${{ github.head_ref || github.run_id }}
+      cancel-in-progress: true
+
+    strategy:
+      fail-fast: false
+      matrix:
+        repo:
+          - 'redaxo/redaxo'
+          - 'yakamara/yform'
+
+    if: github.event.pull_request.draft == false
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: ${{ matrix.repo }}
+
+      - name: Setup PHP
+        id: setup-php
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.1
+          coverage: none
+
+      - name: set redaxo/php-cs-fixer-config to current branch
+        run: composer require redaxo/php-cs-fixer-config:dev-${{ github.head_ref }} --no-scripts --no-update --dev
```

Comment:
Funktioniert das auch, wenn Externe PRs hier im Repo erstellen, die also in einem Fork liegen?
Passiert hier im Repo bisher nicht wirklich. Daher vielleicht irrelevant. Andererseits wäre es aber gerade da auch interessant, die Auswirkungen zu sehen.

Reply:
Ne, fork würde aktuell nicht gehen. Aber glaube nicht, dass das aktuell notwendig ist